ui/lib/apps/InstanceProfiling/pages/Detail.tsx

```diff
@@ -257,6 +155,51 @@ export default function Page() {
     return [newRows, newGroups]
   }, [data])
 
+  const openResult = usePersistFn(
+    async (openAs: string, rec: ActionButtonProps) => {
```

Comment:
Better to call it something like `IRecord`. It is the ActionButton that relying on record's data, instead of the record relying on the action button's data.